### ISSUE TITLE
Logo is not shown

### ISSUE THREAD

**Issue by tikkss** [ruby-gnome/ruby-gnome.github.io#10]
Dear Developers.

Step to reproduce
---

1. open https://ruby-gnome.github.io/

Expect Result
---

<img width="653" alt="image" src="https://user-images.githubusercontent.com/4122070/223552717-3926c283-358b-4841-95c2-b563df81bab0.png">

Actual Result
---

![image](https://user-images.githubusercontent.com/4122070/223551695-49c8a93e-0124-494f-a542-4a32ff91f8d9.png)

Maybe, below diff.

```diff
- https://github.com/ruby-gnome/ruby-gnome/blob/master/gtk3/sample/gtk-demo/ruby-gnome2-logo.png
+ https://raw.githubusercontent.com/ruby-gnome/ruby-gnome/master/gtk3/sample/gtk-demo/ruby-gnome2-logo.png
```

**Comment by kou** [comment]
Good catch!

Could you open a pull request that changes https://github.com/ruby-gnome/ruby-gnome.github.io/blob/main/README.md ?

**Comment by tikkss** [comment]
Yeah! I would like to open a pull request tomorrow.

**Comment by tikkss** [comment]
This logo is now visible when opening https://ruby-gnome.github.io/ .
Could you close this issue?

**Comment by kou** [comment]
Closed.

We should have used "fix #10", "close #10" or something instead of "This fixes the issue #10." in #11 instead.

**Comment by tikkss** [comment]
Thank you for your advice.

I referred to [Linking a pull request to an issue - GitHub Docs](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).
I should have used "Fixes #10." instead of "This fixes the issue #10.".

I would like to link a pull request to an issue using a keyword.
Thanks!
